src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,9 @@
+@mixin day-block($length) {
+  box-sizing: border-box;
```

Comment:
Try setting 'box-sizing: border-box' for the whole project. Maybe put it in the body. Because it seems like your borders are overlapping.